src/app/api/scoreboard/[date]/route.js

```diff
@@ -0,0 +1,15 @@
+import { NextResponse } from 'next/server'
+
+export async function GET(request, { params }) {
+  try {
+    // date must be in format YYYY-M-D
+    const scoreboardURL = `https://statsapi.mlb.com/api/v1/schedule?sportId=1&startDate=${params.date}&endDate=${params.date}`
```

Comment:
`params` returns any data passed in as a dynamic route segment.  In this case since the path for this file is `/api/scoreboard/[date]` we have access to a `date` property which we access by `params.date`.